tests/python/PyUnusedCodeBearTest.py

```diff
@@ -1,19 +1,33 @@
-from queue import Queue
-
 from bears.python.PyUnusedCodeBear import PyUnusedCodeBear
-from tests.LocalBearTestHelper import LocalBearTestHelper
-from coalib.settings.Section import Section
+from tests.LocalBearTestHelper import verify_local_bear
 
+test_file1 = """
+import sys
+import os
+print("Hello World")
+"""
 
-class PyUnusedCodeBearTest(LocalBearTestHelper):
+test_file2 = """
+import sys
+sys.exit()
+"""
 
-    def setUp(self):
-        self.uut = PyUnusedCodeBear(Section('name'), Queue())
+test_file3 = """
+import sys
+import os
+sys.exit()
+import re
+"""
+test_file4 = """
+from sys import exit
+form os import name
 
-    def test_valid(self):
-        self.check_validity(self.uut, ["import sys; sys.do()"])
-        self.check_validity(self.uut, ["a = 2; print(a)"])
+print(name)
+exit()
+"""
+valid = [test_file2, test_file4]
+invalid = [test_file1, test_file3]
 
-    def test_invalid(self):
-        self.check_validity(self.uut, ["import os"], valid=False)
-        self.check_validity(self.uut, ["pass"], valid=False)
+PyUnusedNoReportTest = verify_local_bear(PyUnusedCodeBear,
```

Comment:
Thought of Writing NoReport and Report separately but ended up doing it in one. Will correct that. :+1: